api/v3/GreenpeaceMessageTemplate/Send.php

```diff
@@ -0,0 +1,65 @@
+<?php
+use CRM_Uimods_ExtensionUtil as E;
+
+/**
+ * GreenpeaceMessageTemplate.Send API specification (optional)
+ * This is used for documentation and validation.
+ *
+ * @param array $spec description of fields supported by this API call
+ *
+ * @see https://docs.civicrm.org/dev/en/latest/framework/api-architecture/
+ */
+function _civicrm_api3_greenpeace_message_template_send_spec(&$spec) {
+  $spec['id'] = [
+    'name'         => 'id',
+    'title'        => 'MessageTemplate ID',
+    'type'         => CRM_Utils_Type::T_INT,
+    'api.required' => 1
+  ];
+
+  $spec['to_emails'] = [
```

Comment:
Let's keep the param name as `to_email` (singular) so we don't have to change the parameter name in all tasks.

Reply:
commit e38c4661d58158066094fc5a5c371e41a48b8706